mithril-common/src/chain_observer/pallas_observer.rs

```diff
@@ -861,6 +893,21 @@ mod tests {
         assert_eq!(era, 4);
     }
 
+    #[tokio::test]
+    async fn get_current_era() {
+        let socket_path = create_temp_dir("get_current_era_as_string").join("node.socket");
+        let server = setup_server(socket_path.clone(), 1).await;
+        let client = tokio::spawn(async move {
+            let observer =
+                PallasChainObserver::new(socket_path.as_path(), CardanoNetwork::TestNet(10));
+            observer.get_current_era().await.unwrap().unwrap()
+        });
+
+        let (_, client_res) = tokio::join!(server, client);
+        let era = client_res.expect("Client failed");
+        assert_eq!(era, Era::Mary.to_string());
```

Comment:
It's not obvious where the `Mary` value comes from. 
Can you compute the conversion from the associated `u16` value or update the mock server with the conversion from `Era` to `u16`?

Reply:
You're right 👍, I made the change in this [commit](https://github.com/input-output-hk/mithril/pull/2094/commits/ae5022b44c7806dd426f3a7be6ef4370c3354afb).